extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -211,6 +205,28 @@ class AmpYoutube extends AMP.BaseElement {
     }
   }
 
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    if (mutations['data-videoid'] !== undefined) {
+      this.videoid_ = this.getVideoId_();
+      if (this.iframe_) { // `null` if element hasn't been laid out yet.
```

Comment:
There is short amount of time (actually a good 2 seconds, so not that short :) ) when this.iframe_ is not null but player is not loaded either (meaning commands will be ignored) wrapping the `sendCommand_` in the `this.playerReadyPromise_.then` will queue it up in those cases.